packages/relay/src/lib/eth.ts

```diff
@@ -2032,7 +2032,7 @@ export class EthImpl implements Eth {
         from: EthImpl.zeroAddressHex,
         gasUsed: EthImpl.zeroHex,
         logs: [syntheticLogs[0]],
-        logsBloom: EthImpl.emptyBloom,
+        logsBloom: Utils.createSyntheticLogsBloom(syntheticLogs[0].address, syntheticLogs[0].topics),
```

Comment:
I would personally extract these methods in a separate utility class `LogsBloomUtils` for better organization and modularity. 

In this way, we create a clear separation of concerns, making the `Utils` class focused only on generic helpers, while `LogsBloomUtils` handles all specific operations related to logs bloom filters.